### DISCUSSION TITLE
Suggested default: Follow OS setting whether to use dark/light theme

### DISCUSSION THREAD

**Discussion by albertz** [ghostty-org/ghostty#3168] [Feature Requests, Ideas] [answered]
I really prefer a light theme, and have configured that in my OS, and I want that apps follow this setting, and many apps do. Ghostty does not seem to do that. So I would suggest to change the default.

(I don't really understood how to switch to light theme for Ghostty. Now I just do:
```
background = ffffff
foreground = 282c34
```
I found the doc about theming a bit confusing. I just want to use the default theme, but in light mode. How? But anyway, this is a bit orthogonal to my suggestion here.)

**Comment by gpanders** [comment]
Are you asking for a way to set the _terminal_ background to a light color if the OS theme is light and to a dark color if the OS theme is dark? If so, you want `theme = light:theme-1,dark:theme-2`.

If you're asking about how the window decorations themselves are styled, then you want the `window-theme` option, e.g. `window-theme = system`.

**Reply by albertz** [reply]
> Are you asking for a way to set the _terminal_ background to a light color if the OS theme is light and to a dark color if the OS theme is dark? If so, you want `theme = light:theme-1,dark:theme-2`.

Yes exactly that.

And I am suggesting that this should be the default (to follow the OS theme).

**Reply by albertz** [reply]
Hm, I tried that, then I get:
```
theme "theme-1" not found, tried path "/Users/az/.config/ghostty/themes/theme-1"
theme "theme-1" not found, tried path "/Applications/Ghostty.app/Contents/Resources/ghostty/themes/theme-1"
```

**Reply by albertz** [reply]
Maybe, when I said "theme", this term was a bit wrong. I'm only really caring about the colors. I just want to have the standard default theme.

**Reply by gpanders** [reply]
> Hm, I tried that, then I get:
> 
> ```
> theme "theme-1" not found, tried path "/Users/az/.config/ghostty/themes/theme-1"
> theme "theme-1" not found, tried path "/Applications/Ghostty.app/Contents/Resources/ghostty/themes/theme-1"
> ```

`theme-1` and `theme-2` are just placeholder text. Replace those with the actual theme you want to use. You can view available themes with `ghostty +list-themes`

**Reply by albertz** [reply]
But I don't want to use any specific theme. I just want to get the default. But in light colors when I also have the OS in light.

**Comment by Ianvdl** [comment]
Just to chime in with my 5c. I really like the idea of sane defaults, and the only unexpected side of Ghostty has been that the colors don't automatically switch according to the OS light/dark mode. It would be excellent if I didn't need to add such a config to every installation. 

Thanks for Ghostty!

**Comment by 00-kat** [comment, marked as answer]
This post was first, and by a large margin too, but #5437 makes it much more obvious in what the request is and most constructive discussion on the underlying cause of this not happening by default is in that thread, so I think this should be closed as a duplicate of that (despite that being newer).

**Reply by albertz** [reply]
But I don't just ask for a default light theme. I specifically ask that Ghostty should just follow my system-wide MacOS setting by default. I.e. it should use the light theme by default, not the dark theme, if I set the light theme as the system-wide MacOS theme.

**Reply by 00-kat** [reply]
Mitchell has not decided on what the light theme would be in the first place, so that would be a pre-requisite; though I'm fairly certain that the default would be updated to follow the system when a default light theme exists; ~~perhaps send a note in #2363 (which is the discussion for default configuration changes, by the way)~~ never mind, somebody else already did https://github.com/ghostty-org/ghostty/discussions/2363#discussioncomment-11795512.
